### ISSUE TITLE
Coverage is not reported after v0.1.19

### ISSUE THREAD

**Issue by shouichi** [codecov/codecov-ruby#69]
We don't get coverage reports after updating to v0.1.19 (which starts using v2 API). It seems like uploading is OK but the coverage is never reported. Thanks.

```
  _____          _
 / ____|        | |
| |     ___   __| | ___  ___ _____   __
| |    / _ \ / _\`|/ _ \/ __/ _ \ \ / /
| |___| (_) | (_| |  __/ (_| (_) \ V /
 \_____\___/ \__,_|\___|\___\___/ \_/
                               Ruby-0.1.19
==> Circle CI detected
==> Gzipping contents
==> Uploading reports
    url:   https://codecov.io
    query: token=secret&flags&service=circleci&build=3107&job=0&slug=org%2Frepo&pr&branch=dependabot%2Fbundler%2Fcodecov-0.1.19&commit=896963cf4b95d4ed4cdf2b683af8389780486709
->  Uploading to Codecov
https://codecov.io//upload/v2?token=secret&flags&service=circleci&build=3107&job=0&slug=org%2Frepo&pr&branch=dependabot%2Fbundler%2Fcodecov-0.1.19&commit=896963cf4b95d4ed4cdf2b683af8389780486709
    View reports at https://codecov.io/github/org/repo/commit/896963cf4b95d4ed4cdf2b683af8389780486709
```

**Comment by ShockwaveNN** [comment]
Same here

Here is latest PR without updated codecov
https://github.com/onlyoffice-testing-robot/telegram-github-changes-bot/pull/143
There is codecov report as comment in this PR

This is a bump of codecov iteself and no report as comment
https://github.com/onlyoffice-testing-robot/telegram-github-changes-bot/pull/144

**Comment by AlexWayfer** [comment]
I have a strange HTML response at all:

![image](https://user-images.githubusercontent.com/6510020/86960551-5e3c6f80-c168-11ea-9593-c0e9c215fc6d.png)

https://github.com/AlexWayfer/alt_memery/pull/5

https://cirrus-ci.com/task/6014472798076928

**Comment by AlexWayfer** [comment]
> ### `0.1.18`
>
> - refactor and move to use v2 endpoint

I think, such breaking changes should be in minor version of zero-major, like `0.2.0`. I'm locking at patch-version, and without any changes in Bundle it just stops to work.

**Comment by AlexWayfer** [comment]
`0.1.18` is also broken:

```
Failure/Error: require 'codecov'
LoadError:
  cannot load such file -- colorize
# /usr/local/bundle/gems/codecov-0.1.18/lib/codecov.rb:7:in `require'
# /usr/local/bundle/gems/codecov-0.1.18/lib/codecov.rb:7:in `<top (required)>'
# ./spec/spec_helper.rb:9:in `require'
# ./spec/spec_helper.rb:9:in `<top (required)>'
```

It's fixed here and not in the Changelog: https://github.com/codecov/codecov-ruby/commit/10b2ef171a80243de436e29eb9037695f3832194

So, `'codecov', '~> 0.1.0', '< 0.1.18'` helps.

**Comment by thomasrockhu** [comment]
Hi all, this was clearly a huge mishap on my part. We should not have released the version without testing more thoroughly. We have released `0.1.20` which should patch the issue with the migration to v2. Apologies again and thank you for your patience.
